streamalert/shared/config.py

```diff
@@ -284,8 +268,42 @@ def _validate_config(config):
                            for log_keys in
                            list(config[TopLevelConfigKeys.NORMALIZED_TYPES].values())):
                     raise ConfigError(
-                        'IOC key \'{}\' within IOC type \'{}\' must be defined for at least '
-                        'one log type in normalized types'.format(normalized_key, ioc_type)
+                        "IOC key '{}' within IOC type '{}' must be defined for at least "
```

Comment:
these changes weren't really necessary.. we could have switched the internal `'{}'` quotes to be `"{}"` instead of this alternative.